src/web/css/editor.css

```diff
@@ -625,6 +625,72 @@ div#font-label:hover{
   box-shadow:none;
 }
 
+.highlight-settings {
```

Comment:
A general CSS issue we noticed: there's a rule that gives all `error ul` elements a non-zero padding.  That rule unfortunately supercedes the rule on `ul.inlineCollection` that specifies zero-width padding.  The upshot is that `[list:   empty]` renders with too much whitespace in it.